connectors/golden-tables/src/test/scala/io/delta/golden/GoldenTables.scala

```diff
@@ -104,7 +104,10 @@ class GoldenTables extends QueryTest with SharedSparkSession {
   private implicit class OptimisticTxnTestHelper(txn: OptimisticTransaction) {
     def commitManually(actions: Action*): Long = {
       if (txn.readVersion == -1 && !actions.exists(_.isInstanceOf[Metadata])) {
-        txn.commit(Metadata() +: actions, ManualUpdate)
+        val schema = new StructType()
```

Comment:
A `null` schemaString throws an error in Kernel since the field is non-nullable. Had to regenerate tables with a non-null value.